swagger.yaml

```diff
@@ -3199,8 +3199,6 @@ definitions:
       pull_secret_set:
         type: boolean
         description: True if the pull secret has been added to the cluster.
-      ignition_generator_version:
```

Comment:
Can you please open a ticket about deleting it from the DB? 
Nick added a nice way to do it with https://github.com/openshift/assisted-service/tree/master/internal/migrations

Reply:
added https://issues.redhat.com/browse/MGMT-3095